src/components/CardContainer.jsx

```diff
@@ -5,13 +5,15 @@ class CardContainer extends Component {
   matchDiscussionIds() {
     const { comments, discussions } = this.props;
 
-    let filtered = comments.filter(comment => {
-      return discussions.forEach(discussion => {
-        if (discussion.id === comment.id) {
-          return comment.id
-        }
+    if (discussions) {
```

Comment:
do we still need this condition now that we have App line 106 condition that only renders if (discussions) ?